src/msw/uilocale.cpp

```diff
@@ -96,11 +133,106 @@ wxString wxLocaleIdent::GetName() const
         {
             name << "-" << m_region;
         }
+
+        if (!m_extension.empty())
+        {
+            name << "-" << m_extension;
+        }
+
+        if (!m_sortorder.empty())
+        {
+            name << "_" << m_sortorder;
+        }
     }
 
     return name;
 }
 
+// ----------------------------------------------------------------------------
+// Standard C wxUILocale implementation for MSW
+// ----------------------------------------------------------------------------
+
+class wxUILocaleImplStdC : public wxUILocaleImpl
+{
+public:
+
+    // Create object corresponding to the given locale, return NULL if not
+    // supported.
+    static wxUILocaleImplStdC* Create()
+    {
+        return new wxUILocaleImplStdC();
+    }
+
+
+    ~wxUILocaleImplStdC() wxOVERRIDE
+    {
+    }
+
+    void Use() wxOVERRIDE
+    {
+        // TODO: Should method Use() set a UI locale somehow?
```

Comment:
Without an `LCID` or a locale name it can't do anything, so we shouldn't leave this "TODO" comment, it could never be addressed anyhow.

Reply:
I remove the comment.